gw_eccentricity/eccDefinition.py

```diff
@@ -1525,9 +1525,13 @@ def get_t_average_for_orbit_averaged_omega22(self):
         return [t_for_orbit_averaged_omega22,
                 sorted_idx_for_orbit_averaged_omega22]
 
-    def get_orbit_averaged_omega22_at_pericenters(self):
+    def get_orbit_averaged_omega22_between_pericenters(self):
         """Get orbital average of omega22 between two consecutive pericenters.
 
+        Given N pericenters at times t[i], i=0...N-1, this function
+        returns a np.array of length N-1, where result[i] is the frequency
+        averaged over [t[i], t[i+1]].
```

Comment:
Not clear enough. Point to the equation in the paper, and say what time this average is associated with.

Reply:
Done

Reply:
Thanks @vijayvarma392 for making my suggestion even more precise, and thanks @md-arif-shaikh for implementing it.